<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLidar</class>
 <widget class="QDialog" name="DialogLidar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UART</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_getInfo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_getHealth">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Health</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stopScan">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_MoterEn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Moter</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_IntensityON">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Inten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_IntensityOFF">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Blind</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loopTest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>测试开始</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loopStop">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>串口</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_ID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black; 
color: rgb(0, 0, 0);
background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>20</y>
     <width>31</width>
     <height>181</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>20</width>
     <height>191</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_uart">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 215, 207);
background-color: rgb(211, 215, 207);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>清零</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>49</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Narwel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>184</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_6">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>  通过</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>  不通过</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>光强切换</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumber_switch_Y">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; color:  red; background: silver;</string>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLCDNumber" name="lcdNumber_switch_N">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; color:  red; background: silver;</string>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>光强检测</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="lcdNumber_Intensity_Y">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; color:  red; background: silver;</string>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLCDNumber" name="lcdNumber_Intensity_N">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; color:  red; background: silver;</string>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>点数盲区</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLCDNumber" name="lcdNumber_Blind_Y">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; color:  red; background: silver;</string>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLCDNumber" name="lcdNumber_Blind_N">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; color:  red; background: silver;</string>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_8</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_stopScan</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_open</zorder>
  <zorder>pushButton_getInfo</zorder>
  <zorder>pushButton_getHealth</zorder>
  <zorder>pushButton_MoterEn</zorder>
  <zorder>pushButton_IntensityON</zorder>
  <zorder>pushButton_IntensityOFF</zorder>
  <zorder>pushButton_loopTest</zorder>
  <zorder>pushButton_loopStop</zorder>
  <zorder>lcdNumber_ID</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>lcdNumber_uart</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
